Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nikolaik/python-nodejs:python3.8-nodejs12-alpine
+FROM https://registry.hub.docker.com/r/vukomir/aws-sam-cli
```

Comment:
I don't think you need the full url, `vukomir/aws-sam-cli:latest` should suffice

Reply:
Ah, right, sorry about that part!